examples/plot_classification_methods.py

```diff
@@ -0,0 +1,132 @@
+"""
+Classification methods.
+
+==================================
+
+Shows a comparison between different classification methods.
```

Comment:
Several things wrong with the example:
- Almost no explanation of the methods/why they have been chosen.
- The plotted images don't say much.
- No comparison of the accuracy of each method.
- No conclusions at all.